app/view/account_transaction_details.py

```diff
@@ -28,26 +28,47 @@ def post(self):
                 response = ResponseGenerator(data={}, message=result, success=False,
                                              status=status.HTTP_400_BAD_REQUEST)
                 return response.error_response()
-            account = BankAccount.query.filter(BankAccount.id == account_transaction_data['bank_account_id']).first()
-            transaction_type = TransactionType.query.filter(TransactionType.id == account_transaction_data['transaction_type_id']).first()
+            account = BankAccount.query.filter(
+                BankAccount.id == account_transaction_data['bank_account_id']).first()
+            transaction_type = TransactionType.query.filter(
+                TransactionType.id == account_transaction_data['transaction_type_id']).first()
```

Comment:
For such queries you can learn about `joins`, and use join queries here.

Reply:
Yes, I have seen about joins now, but joins requires a relationship between the tables and the BankAccount and TransactionType tables does not share any relationship.